### PR TITLE
파일 업로드 요청합니다

### PR DESCRIPTION
**Pull request by Sionruu**

### CODE REVIEW

**Comment by BluePinetree** [comment]
_Commented on notebook `week1/김민수/mlp.ipynb` Cell 12 Line 3_
```python
opt = optimizers.Adam(learning_rate=0.0005)
model1.compile(
    loss="mae", optimizer=opt, metrics=["mape"]
```
보스턴 집값 예측 데이터셋을 가져와 기존의 분류 문제를 회귀 문제로 변환하였습니다.
이에 알맞게 손실함수를 편균절대오차(Mean Absolute Error)로 변경하였으며, 평가 척도를 Mean Absolute Percentage Error를 사용하여 성능을 지속적으로 추적할 수 있도록 하였습니다.
아주 좋습니다.

_Respond and view the context [here](https://gitnotebooks.com/SPAI-Lab/GenerativeAI_Study/pull/19#comment-1060)._

**Comment by BluePinetree** [comment]
_Commented on notebook `week1/김민수/mlp.ipynb` Cell 18 Line 4_
```python
x = np.arange(0, len(preds))
plt.plot(x, preds, label='prediction', alpha=0.5)
plt.plot(x, y_test, label='test', alpha=0.5)
plt.legend()
```
실제 집값과 예측한 집값의 차이를 확인하기 위해 시각화를 진행하였군여. 좋습니다.
다만, x축과 y축이 무엇을 의미하는지 label로서 명시를 해주었으면 더 좋은 시각화가 되었을 듯 합니다.

_Respond and view the context [here](https://gitnotebooks.com/SPAI-Lab/GenerativeAI_Study/pull/19#comment-1061)._

**Comment by BluePinetree** [comment]
_Commented on notebook `week2/김민수/cnn_custom_dataset1.ipynb` Cell 11 Line 32_
```python
x = layers.Flatten()(x)
x = layers.Dense(32)(x)
x = layers.BatchNormalization()(x)
x = layers.LeakyReLU()(x)
x = layers.Dropout(rate=0.2)(x)

x = layers.Dense(1)(x)
output_layer = layers.Activation("softmax")(x)
```
스터디 시간에도 언급하였듯이, softmax 활성화 함수는 multi-class classification 문제에만 사용할 수 있는 문제입니다. 모델의 출력이 2개 이하인 이진 분류 문제에는 해당 함수를 사용하였을 시에 올바로게 동작하지 않을 가능성이 높습니다.

따라서 이진 분류 문제에는 sigmoid 활성화 함수를 사용하는 것이 더욱 나은 성능을 보여줄 수 있을 것으로 기대됩니다.

직접 실험해보시고 그 결과를 공유해주시면 모두에게 좋은 경험이 될것으로 기대됩니다.

_Respond and view the context [here](https://gitnotebooks.com/SPAI-Lab/GenerativeAI_Study/pull/19#comment-1062)._

**Review by BluePinetree — APPROVED** [review]
MLP 문제에서는 분류 문제를 회귀로, CNN 문제에서는 개와 고양이 이진 분류 데이터셋으로, 그리고 Diabetes 문제에서는 심층 신경망 모델을 사용하여 훈련을 진행해 보았군여. 멋집니다.
